pkgs/tools/misc/urn-timer/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, unstableGitUpdater
+, xxd
+, pkg-config
+, imagemagick
+, rsync
+, wrapGAppsHook
+, gtk3
+, jansson
+}:
+
+stdenv.mkDerivation {
+  pname = "urn-timer";
+  version = "unstable-2017-08-20";
+
+  src = fetchFromGitHub {
+    owner = "3snowp7im";
+    repo = "urn";
+    rev = "246a7a642fa7a673166c1bd281585d0fc22e75b2";
+    sha256 = "0pdf573hgc6qd317fh5275y772q1wpqq81qzgl5gxclz6kh2kczs";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "stop-hardcoding-prefix";
+      url = "https://github.com/3snowp7im/urn/pull/50.patch";
+      sha256 = "1xdkylbqlqjwqx4pb9v1snf81ag7b6q8vybirz3ibsv6iy79v9pk";
+    })
+    (fetchpatch {
+      name = "create-installation-directories";
+      url = "https://github.com/3snowp7im/urn/pull/53.patch";
+      sha256 = "0jjhcz4n8bm3hl56rvjzkvxr6imc05qlyavzjrlafa19hf036g4a";
+    })
+  ];
+
+  nativeBuildInputs = [
+    xxd
+    pkg-config
+    imagemagick
+    rsync
```

Comment:
Is rsync only used once? Might be worth it to replace it with cp.

Reply:
done